### DISCUSSION TITLE
Contribute with a custom file extension and sintax highlight to be able to use another debugger extension

### DISCUSSION THREAD

**Discussion by microhobby** [microsoft/vscode-discussions#2380] [Extension Development QnA] [answered]
I have a case where I have a custom file extension, a custom syntax for this file extension, but it is possible to use the debugger from Python Debugpy extension. There is a way to contribute my file extension to be able to add breakpoints but the debugger will be implemented by another extension?

**Comment by isidorn** [comment, marked as answer]
Not possible. As far as I know.

**Comment by RedCMD** [comment]
ask the python extension to support your language/file extension
